ProyectoFinal/src/visual/RegEmpresa.java

```diff
@@ -53,211 +62,375 @@ public static void main(String[] args) {
+			lblNewLabel.setBounds(33, 32, 94, 14);
+			panel.add(lblNewLabel);
+			try{
+				txtFRNC = new JFormattedTextField(Utils.getMaskCedula());
+				txtFRNC.setForeground(Color.BLACK);
+				txtFRNC.setToolTipText("");
+				txtFRNC.setBounds(120, 29, 146, 20);
+				panel.add(txtFRNC);
+			}catch(Exception ex){}
 			{
-				JLabel label = new JLabel("Direcci\u00F3n:");
-				label.setBounds(23, 72, 195, 14);
-				panel_1.add(label);
+				JLabel lblNewLabel_1 = new JLabel("Tipo de empresa:");
+				lblNewLabel_1.setBounds(382, 115, 98, 14);
+				panel.add(lblNewLabel_1);
 			}
 			{
-				textField_3 = new JTextField();
-				textField_3.setColumns(10);
-				textField_3.setBounds(161, 70, 179, 20);
-				panel_1.add(textField_3);
+				cbxTipo = new JComboBox();
+				cbxTipo.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Micro", "Mediana", "Peque\u00F1a ", "Grande", "Emergente"}));
+				cbxTipo.setBounds(502, 112, 156, 20);
+				panel.add(cbxTipo);
 			}
 			{
-				JLabel label = new JLabel("Provincia:");
-				label.setBounds(372, 31, 195, 14);
-				panel_1.add(label);
+				JLabel lblNewLabel_2 = new JLabel("Sector:");
+				lblNewLabel_2.setBounds(382, 73, 98, 14);
+				panel.add(lblNewLabel_2);
 			}
 			{
-				JLabel label = new JLabel("Pa\u00EDs:");
-				label.setBounds(372, 72, 195, 14);
-				panel_1.add(label);
+				JLabel lblNombre = new JLabel("Nombre Comercial:");
+				lblNombre.setBounds(382, 32, 141, 14);
+				panel.add(lblNombre);
 			}
 			{
-				textField_4 = new JTextField();
-				textField_4.setColumns(10);
-				textField_4.setBounds(443, 70, 195, 20);
-				panel_1.add(textField_4);
-			}
+				txtRubro = new JTextField();
+				txtRubro.setColumns(10);
+				txtRubro.setBounds(120, 112, 146, 20);
+				panel.add(txtRubro);
+			}			
 			{
-				textField_5 = new JTextField();
-				textField_5.setColumns(10);
-				textField_5.setBounds(443, 31, 195, 20);
-				panel_1.add(textField_5);
+				txtNombreComercial = new JTextField();
+				txtNombreComercial.setColumns(10);
+				txtNombreComercial.setBounds(502, 29, 156, 20);
+				panel.add(txtNombreComercial);
 			}
-		}
-		{
-			JPanel panel_1 = new JPanel();
-			panel_1.setLayout(null);
-			panel_1.setBorder(new TitledBorder(UIManager.getBorder("TitledBorder.border"), "Datos de Contacto", TitledBorder.LEADING, TitledBorder.TOP, null, new Color(0, 0, 0)));
-			panel_1.setBounds(10, 160, 687, 117);
-			panel.add(panel_1);
 			{
-				JLabel lblNombre_1 = new JLabel("Nombre:");
-				lblNombre_1.setBounds(23, 31, 195, 14);
-				panel_1.add(lblNombre_1);
+				cbxSector = new JComboBox();
+				cbxSector.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Industrial", "Agricultura ", "Alimentaci\u00F3n", "Comercio ", "Construcci\u00F3n", "Educaci\u00F3n ", "Hoteler\u00EDa", "Medios de comunicaci\u00F3n ", "Miner\u00EDa ", "Petrolero ", "Telecomunicaciones ", "Salud ", "Financieros", "P\u00FAblico ", "Silvicultura ", "Textil ", "Tecnol\u00F3gico", "Transporte"}));

```

Comment:
```suggestion
				cbxSector.setModel(new DefaultComboBoxModel(new String[] {"<Seleccione>", "Industrial", "Agricultura", "Alimentaci\u00F3n", "Comercio", "Construcci\u00F3n", "Educaci\u00F3n", "Hoteler\u00EDa", "Medios de comunicaci\u00F3n", "Miner\u00EDa", "Petrolero", "Telecomunicaciones", "Salud", "Financieros", "P\u00FAblico", "Silvicultura", "Textil", "Tecnol\u00F3gico", "Transporte"}));

```

Los datos tenían espacios, esto puede ser problemático en las comparaciones.